### PR TITLE
Added documentation blacklisting notes

### PR DESCRIPTION
**Pull request by vignesh-sankaran**
Added notes regarding [documentation domain blacklisting](https://github.com/rust-lang/crates.io/issues/626) on crates.io

### CODE REVIEW

**Comment by matklad** [comment]
r? @carols10cents 

I have no idea what's going on here :)

**Comment by carols10cents** [comment]
This can't go in until the implementation in https://github.com/rust-lang/crates.io/pull/816 does and I'm super behind reviewing crates.io PRs. Sorry for the delay!

File: src/doc/manifest.md [hunk 1]
```diff
@@ -48,6 +48,11 @@ This field specifies a URL to a website hosting the crate's documentation.
 If no URL is specified in the manifest file, [crates.io][cratesio] will
 automatically link your crate to the corresponding [docs.rs][docsrs] page.
 
+Documentation links from specific domainss are blacklisted. Currently, any link
```
Comment (carols10cents) [inline src/doc/manifest.md]: Typo: "domainss" should be "domains :)

File: src/doc/manifest.md [hunk 2]
```diff
@@ -48,6 +48,11 @@ This field specifies a URL to a website hosting the crate's documentation.
...
+from rust-ci.org will be blacklisted since this domain is no longer used for
+legitimate documentation hosting. URLs from this domain will have no reported
+documentation link on crates.io.
```
Comment (carols10cents) [inline src/doc/manifest.md]: Actually, because of the docs.rs integration (described on line 48 above this), they might have docs.rs links instead... could we reword this as "Documentation URLs from blacklisted hosts will not appear on crates.io, and may be replaced by docs.rs links." Or something similar?

File: src/doc/manifest.md [hunk 3]
```diff
@@ -48,6 +48,11 @@ This field specifies a URL to a website hosting the crate's documentation.
 If no URL is specified in the manifest file, [crates.io][cratesio] will
 automatically link your crate to the corresponding [docs.rs][docsrs] page.
 
+Documentation links from specific domainss are blacklisted. Currently, any link
+from rust-ci.org will be blacklisted since this domain is no longer used for
```
Comment (carols10cents) [inline src/doc/manifest.md]: Could we reorganize this paragraph a bit to make it easier to add more hosts in the future? So instead of having rust-ci.org mentioned in the paragaph, have the paragraph end with: 

> The blacklist currently consists of:
>   - rust-ci.org

so that adding more hosts just entails adding another bullet?

**Comment by carols10cents** [comment]
The change described has been deployed to crates.io!

**Comment by vignesh-sankaran** [comment]
Thanks for the suggestions @carols10cents, I'll push the updated docs in a couple of hours or so :)

File: src/doc/manifest.md (hunk 2 above)
Comment (vignesh-sankaran) [inline src/doc/manifest.md]: Yep on it :)

**Comment by vignesh-sankaran** [comment]
@carols10cents How does the documentation look now?

**Comment by carols10cents** [comment]
lgtm! thank you!!

@bors: r+

**Comment by carols10cents** [comment]
lol oops! @matklad I'm not allowed, but this PR looks good to go :)
